Changed a recommendation for custom events name casing (#754)
vuejs/docs-next: src/guide/component-custom-events.md

## Event Names

Like components and props, event names provide an automatic case transformation. If you emit an event from the child component in camel case, you will be able to add a kebab-cased listener in the parent:

```js
this.$emit('myEvent')
```

```html
<my-component @my-event="doSomething"></my-component>
```

> Similarly to [props casing](/guide/component-props.html#prop-casing-camelcase-vs-kebab-case), we recommend using kebab-cased event listeners when you are using in-DOM templates. If you're using string templates, this limitation does not apply.

Reviewer: ```suggestion
As with [props casing](/guide/component-props.html#prop-casing-camelcase-vs-kebab-case), we recommend using kebab-cased event listeners when you are using in-DOM templates. If you're using string templates, this limitation does not apply.
```

At first reading, the use of *'Similarly to'* derailed me a bit. To me it didn't sound right. My instinctive reaction was to change it to *'Similar to'*.

However, I spent some time researching this online and, with hindsight, I think *'Similarly to'* probably is technically correct. That said, plenty of native speakers on those same forums seemed to agree that it sounds wrong, even if it isn't. A recurring suggestion was to reword the sentence altogether, which is what I've attempted.
